utils/envvars/parser.go

```diff
@@ -163,22 +161,22 @@ func parseStringDuration(value string) (time.Duration, error) {
 
 // GetSAASHost returns SaaS host env variable value if it's present and valid.
 // Otherwise returns defaultSaaSHost.
-func GetSAASHost(fallbackEnv string) (string, error) {
+func GetSAASHost() (string, error) {
 	name, v := envSaaSHost, os.Getenv(envSaaSHost)
 	if v == "" {
-		name, v = fallbackEnv, os.Getenv(fallbackEnv)
-	}
-	if v == "" {
-		logrus.Debugf("Using default SaaS host %q.", defaultSaaSHost)
+		logrus.Infof("Using default SaaS host %q.", defaultSaaSHost)
 		return defaultSaaSHost, nil
 	}
+	if strings.HasPrefix(v, ":") {
+		return "", fmt.Errorf("environment variable %q has invalid format %q. Expected host:[port]", name, v)
```

Comment:
I suppose the correct format is `host[:port]`

Reply:
done thanks